Add configuration option for slather action

thasegaw opened the pull request:
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Description
[slather-2.4.1](https://github.com/SlatherOrg/slather/releases/tag/v2.4.1) is added `configuration` option. 

### Motivation and Context
Support slather-2.4.1

Commit:
support configuration option for slather

mgrebenets commented:
@thasegaw 

I'm not sure how, but is it possible to have some kind of version check for this new option?
I means it should check that `slather` is installed first (which I believe is already the case), but then it needs to make sure that version is at least `2.4.1` if `configuration` option is used.

Probably that could be part of configuration option verify block.

thasegaw committed:
Check whether slather is available version or not

thasegaw commented:
@mgrebenets 
I added the verification whether slather version is at least 2.4.1 or not. I added it in` self.validate_params!` because I have to check `use_bundle_exec` parameter. Colud you let me know, If you have good idea. Thanks

thasegaw commented:
@KrauseFx Would you check this RP? I want to use this option.

KrauseFx commented on fastlane/lib/fastlane/actions/slather.rb:
```diff
@@ -52,7 +53,22 @@ def self.has_config_file
       end
 
+      def self.slather_version(params)
+        command = []
```
Instead of shelling out to receive the version number, can't we do something like
```ruby
require 'slather'
Slather::VERSION
```

Commit:
use Slather::VERSION instead of shelling out

Commit:
Merge remote-tracking branch 'upstream/master' into slather_configuration

KrauseFx reviewed (approved):
Awesome, thanks for updating the PR, and thanks for working on this 🚀 This will be in the next release 👍
